queries/boundaries.jinja2

```diff
@@ -84,6 +85,7 @@ SELECT
   'yes' AS maritime_boundary,
   NULL AS admin_level,
   NULL AS boundary_type,
+  NULL AS kind,
```

Comment:
As Matt points out we're doing the [kind assignment](https://github.com/mapzen/TileStache/blob/integration-1/TileStache/Goodies/VecTiles/transform.py#L361-L366) in Python and not in the SQL query. That mostly seems to be working for existing boundary features (like country, state, etc), and if it needs refinement let's file a different issue for it.